pages/common/twm.md

```diff
@@ -0,0 +1,24 @@
+# twm
```

Comment:
Should this be moved to the `linux` platform directory?

Reply:
It works in Mac and Linux, so I wasn't sure. But I guess it could be in the Mac and Linux directories instead. 🤷🏻

Reply:
I invoke to @sbrl to know what to do. lol

Reply:
If it works on more than 1 platform, then it goes in `common` @navarroaxel :slightly_smiling_face: